Formula/h/hashlink.rb

```diff
@@ -46,17 +46,35 @@ class Hashlink < Formula
   end
 
   def install
+    # NOTE: This installs lib/*.hdll files which would be audited by `--new`.
+    # These appear to be renamed shared libraries specifically used by HashLink.
+    args = ["PREFIX=#{prefix}"]
+
     if OS.mac?
       # make file doesn't set rpath on mac yet
-      system "make", "PREFIX=#{libexec}", "EXTRA_LFLAGS=-Wl,-rpath,#{libexec}/lib"
+      args << "EXTRA_LFLAGS=-Wl,-rpath,#{lib}"
```

Comment:
```suggestion
      args << "EXTRA_LFLAGS=-Wl,-rpath,#{rpath}"
```
Does this not work?

Reply:
It probably does (RPATH for `bin/hl`) but didn't try it out. Can try locally and update PR.

Reply:
We should use `rpath` where we can since that produces an `LC_RPATH` command that `brew` doesn't need to relocate (faster bottle pour times).